src/theme/components/multiSelect.ts

```diff
@@ -1,3 +1,17 @@
+import { createMultiStyleConfigHelpers } from "@chakra-ui/styled-system";
+
+// This function creates a set of function that helps us
+// create multipart component styles.
+const { defineMultiStyleConfig, definePartsStyle } =
+  createMultiStyleConfigHelpers([
+    "base",
+    "accordionStyles",
+    "buttonTextLabel",
+    "menuChildren",
+    "menuSearchInputBox",
```

Comment:
I know these are the names that were already in there, but for consistency with the docs, change the prefix from `menu` to `list`.

```
"listChildren",
"listSearchInputbox",
```